src/sentry/interfaces/stacktrace.py

```diff
@@ -141,7 +141,7 @@ def is_newest_frame_first(event):
 
 
 def is_url(filename):
-    return filename.startswith(('file:', 'http:', 'https:'))
+    return filename.startswith(('file:', 'http:', 'https:', 'applewebdata:'))
```

Comment:
Not thrilled about this but worried about doing more damage than benefit by matching things that already affect grouping